src/Core/src/Handlers/DatePicker/DatePickerHandler.iOS.cs

```diff
@@ -127,6 +133,16 @@ void SetVirtualViewDate()
 			VirtualView.Date = DatePickerDialog.Date.ToDateTime().Date;
 		}
 
+		void OnCultureChanged()
```

Comment:
Culture Changed is never notified , As the runtime changes doesnot triggers this method